tests/test_plugins/test_microwave.py

```diff
@@ -1,5 +1,9 @@
 import pytest
```

Comment:
can you add a module level docstring and docstrings for the individual tests?

Also you may want to run `ruff tests/test_plugins/test_microwave` as we'll start `ruff`ing test files from 2.7.0

Reply:
careful about ruffing the whole test directory or you'll get a lot of changes / errors, many of which are already covered #1707